File: samples/server/petstore/java-helidon-server/se/src/main/java/org/openapitools/server/model/EnumArrays.java
package org.openapitools.server.model;

import com.fasterxml.jackson.annotation.JsonCreator;
import com.fasterxml.jackson.annotation.JsonValue;
import java.util.ArrayList;
import java.util.Arrays;
import java.util.List;



public class EnumArrays   {


    /**
    * Gets or Sets justSymbol
    */
    public enum JustSymbolEnum {
        GREATER_THAN_OR_EQUAL_TO(">="),
        DOLLAR("$");

        private String value;

        JustSymbolEnum(String value) {
            this.value = value;
        }

        @JsonValue
        public String getValue() {
            return value;
        }

        @Override
        public String toString() {
            return String.valueOf(value);
        }


        @JsonCreator
        public static JustSymbolEnum fromValue(String text) {
            for (JustSymbolEnum b : JustSymbolEnum.values()) {
                if (String.valueOf(b.value).equals(text)) {
                    return b;
                }
            }
            throw new IllegalArgumentException("Unexpected value '" + text + "'");
        }
    }


    private JustSymbolEnum justSymbol;

    /**
    * Gets or Sets arrayEnum
    */
    public enum ArrayEnumEnum {
        FISH("fish"),
        CRAB("crab");

        private String value;

        ArrayEnumEnum(String value) {
            this.value = value;
        }

        @JsonValue
        public String getValue() {
            return value;
        }

        @Override
        public String toString() {
            return String.valueOf(value);
        }


        @JsonCreator
        public static ArrayEnumEnum fromValue(String text) {
            for (ArrayEnumEnum b : ArrayEnumEnum.values()) {
                if (String.valueOf(b.value).equals(text)) {
                    return b;
                }
            }
            throw new IllegalArgumentException("Unexpected value '" + text + "'");
        }
    }


    private List<ArrayEnumEnum> arrayEnum;

    /**
     * Default constructor.
     */
    public EnumArrays() {
    // JSON-B / Jackson
    }

    /**
     * Create EnumArrays.
     *
     * @param justSymbol justSymbol
     * @param arrayEnum arrayEnum
     */
    public EnumArrays(
        JustSymbolEnum justSymbol, 
        List<ArrayEnumEnum> arrayEnum
    ) {
        this.justSymbol = justSymbol;
        this.arrayEnum = arrayEnum;
    }



    /**
     * Get justSymbol
     * @return justSymbol
     */
    public JustSymbolEnum getJustSymbol() {
        return justSymbol;
    }

    public void setJustSymbol(JustSymbolEnum justSymbol) {
        this.justSymbol = justSymbol;
    }

    /**
     * Get arrayEnum
     * @return arrayEnum
     */
    public List<ArrayEnumEnum> getArrayEnum() {
        return arrayEnum;
    }

    public void setArrayEnum(List<ArrayEnumEnum> arrayEnum) {
        this.arrayEnum = arrayEnum;
    }

    /**
      * Create a string representation of this pojo.
    **/
    @Override
    public String toString() {
        StringBuilder sb = new StringBuilder();
        sb.append("class EnumArrays {\n");
        
        sb.append("    justSymbol: ").append(toIndentedString(justSymbol)).append("\n");
        sb.append("    arrayEnum: ").append(toIndentedString(arrayEnum)).append("\n");
        sb.append("}");
        return sb.toString();
    }

    /**
     * Convert the given object to string with each line indented by 4 spaces
     * (except the first line).
    */
    private static String toIndentedString(Object o) {
        if (o == null) {
          return "null";
        }
        return o.toString().replace("\n", "\n    ");
    }
}

File: samples/server/petstore/kotlin-springboot-request-cookie/src/main/kotlin/org/openapitools/model/EnumArrays.kt
package org.openapitools.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import com.fasterxml.jackson.annotation.JsonValue
import jakarta.validation.constraints.DecimalMax
import jakarta.validation.constraints.DecimalMin
import jakarta.validation.constraints.Email
import jakarta.validation.constraints.Max
import jakarta.validation.constraints.Min
import jakarta.validation.constraints.NotNull
import jakarta.validation.constraints.Pattern
import jakarta.validation.constraints.Size
import jakarta.validation.Valid
import io.swagger.v3.oas.annotations.media.Schema

/**
 * 
 * @param justSymbol 
 * @param arrayEnum 
 */
data class EnumArrays(

    @Schema(example = "null", description = "")
    @get:JsonProperty("just_symbol") val justSymbol: EnumArrays.JustSymbol? = null,

    @Schema(example = "null", description = "")
    @get:JsonProperty("array_enum") val arrayEnum: kotlin.collections.List<EnumArrays.ArrayEnum>? = null
) {

    /**
    * 
    * Values: greaterThanEqual,dollar
    */
    enum class JustSymbol(val value: kotlin.String) {

        @JsonProperty(">=") greaterThanEqual(">="),
        @JsonProperty("$") dollar("$")
    }

    /**
    * 
    * Values: fish,crab
    */
    enum class ArrayEnum(val value: kotlin.String) {

        @JsonProperty("fish") fish("fish"),
        @JsonProperty("crab") crab("crab")
    }

}

